notebook/static/notebook/js/actions.js

```diff
@@ -437,7 +437,18 @@ define(function(require){
                 $('.header-bar').toggle();
                 events.trigger('resize-header.Page');
             }
-        }
+        },
+        'escape': {
```

Comment:
escape will be the name in the CP I woudl be more verbose. actually: `close-pager-or-unmark-all-cells
`